### ISSUE TITLE
import error on M1 macbook

### ISSUE THREAD

**Issue by leahwangibm** [ibmdb/python-ibmdb#930]
Hi, 

I followed the steps from **ibm_db installation on MacOS M1/M2 Chip System (arm64 architecture)** section in **INSTALL.md** but still getting below error when I tried to import the package. 

```
bash-3.2$ arch -x86_64 pip3 install ibm_db
Collecting ibm_db
  Using cached ibm_db-3.2.3-cp39-cp39-macosx_10_15_x86_64.whl (24.0 MB)
Installing collected packages: ibm_db
Successfully installed ibm_db-3.2.3
WARNING: You are using pip version 22.0.4; however, version 24.0 is available.
You should consider upgrading via the '/Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9 -m pip install --upgrade pip' command.
bash-3.2$
bash-3.2$ python3
Python 3.9.11 (v3.9.11:2de452f8bf, Mar 16 2022, 10:34:36)
[Clang 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ibm_db
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: dlopen(/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/ibm_db.cpython-39-darwin.so, 0x0002): Symbol not found: __ZNKSt8__detail20_Prime_rehash_policy14_M_need_rehashEmmm
  Referenced from: <803DFCA0-2956-3B4C-B633-4BCFBAA8B43A> /Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/clidriver/lib/libdb2.dylib
  Expected in:     <A6006A59-BA45-35D6-8D8E-C49F5BA1EDF7> /usr/lib/libstdc++.6.dylib
>>>

```

Could anyone help me with this? Or any other ways to use python to connect to Db2 on Z/OS would also be appreciated!!

- Operating System Name: Sonoma 14.3.1
- db2level output from clidriver if in use: 
- Target Db2 Server Version: 
- Python Version: 3.9.11 (as instructed)
- ibm_db version: ibm_db-3.2.3
- For non-Windows, output of below commands:
  uname Darwin
  uname -m x86_64
- Value of below environment variables if set:
  IBM_DB_HOME:
  PATH:
  LIB/LD_LIBRARY_PATH/DYLD_LIBRARY_PATH:

- Test script to reproduce the problem.
- For installation related issue, complete output of `pip install ibm_db` command.

Thanks in advance!!

**Comment by bchoudhary6415** [comment]
Hello @leahwangibm 
You are getting Symbol not found error.
Please check the section 4 - https://github.com/ibmdb/python-ibmdb/blob/master/INSTALL.md#-4-troubleshooting-post-install-errors, the point number 4.2

Please try to execute all the steps of point number 4.2.

Thanks

**Comment by bimalkjha** [comment]
@leahwangibm We have released `ibm_db v3.2.5` with native installation support for MacOS Silicon Chip system and no need to use the emulated environment with x86_64 python on M1 chip system. Please install latest version of ibm_db and verify. Thanks.
